editor/src/utils/canvas-react-utils.ts

```diff
@@ -269,15 +290,18 @@ const mangleExoticType = Utils.memoize(
           const originalFunction = p.children
           children = function (...params: any[]) {
             const originalResponse = originalFunction(...params)
-            return attachDataUidToRoot(originalResponse, uids)
+            return attachDataUidToRoot(originalResponse, uids, paths)
           }
         } else {
           const uidsToPass = uidsWithoutExoticUID(uids)
+          const pathsToPass = uidsWithoutExoticUID(paths)
```

Comment:
I think this line might be wrong and might be causing some of the weirdness in the tests, so it's worth checking that this is behaving correctly with contexts and the like

Reply:
Actually ignore this :point_up: I think this should still result in it working the same way as before...